src/createEngineStream.ts

```diff
@@ -0,0 +1,44 @@
+import { Duplex } from 'readable-stream';
+import { JsonRpcEngine, JsonRpcRequest } from 'json-rpc-engine';
+
+interface EngineStreamOptions {
+  engine: JsonRpcEngine;
+}
+
+/**
+ * Takes a JsonRpcEngine and returns a Duplex stream wrapping it.
+ * 
```

Comment:
```suggestion
 *
```
lint error- extra space